graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -30,7 +30,7 @@
     StringType,
     LongType,
 )
-from scipy.special import erfinv
+from scipy.special import erfinv  # pylint: disable=no-name-in-module
```

Comment:
Why do we need this disable? I suppose the erfinv should be in the scipy.special

Reply:
It should but for some reason it's not picked up by pylint